commons/src/main/java/io/aiven/kafka/connect/common/source/input/ByteArrayTransformer.java

```diff
@@ -18,33 +18,38 @@
 
 import java.io.IOException;
 import java.io.InputStream;
-import java.nio.charset.StandardCharsets;
 import java.util.Arrays;
-import java.util.Map;
 import java.util.function.Consumer;
 
 import org.apache.kafka.common.config.AbstractConfig;
+import org.apache.kafka.connect.data.Schema;
 import org.apache.kafka.connect.data.SchemaAndValue;
 
+import io.aiven.kafka.connect.common.OffsetManager;
+
 import org.apache.commons.io.IOUtils;
 import org.apache.commons.io.function.IOSupplier;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-public class ByteArrayTransformer extends Transformer<byte[]> {
+/**
+ * A transformer that reads plain bytes from the input stream.
+ */
+public class ByteArrayTransformer extends Transformer {
+    /** The logger for this transformer */
     private static final Logger LOGGER = LoggerFactory.getLogger(ByteArrayTransformer.class);
-
+    /** The maximum record size this transform will read. If more data is sent then the record is ignored. */
     private static final int MAX_BUFFER_SIZE = 4096;
 
     @Override
-    public void configureValueConverter(final Map<String, String> config, final AbstractConfig sourceConfig) {
-        // For byte array transformations, ByteArrayConverter is the converter which is the default config.
+    public Schema getKeySchema() {
+        return null;
```

Comment:
I know this is because the ByteArrayTransformer isgnores schemas, but can you add a comment there to explain it to any future dev?